enterprise_gateway/services/processproxies/container.py

```diff
@@ -200,12 +201,19 @@ async def confirm_remote_startup(self) -> None:
 
             container_status = self.get_container_status(str(i))
             if container_status:
-                if self.assigned_host != "":
-                    ready_to_connect = await self.receive_connection_info()
-                    self.pid = (
-                        0  # We won't send process signals for kubernetes lifecycle management
+                self.log.debug(f"Received startup status: {container_status}")
```

Comment:
I don't think this is providing much either.

Reply:
it's debug log, and helps to try to understand the area that things are happening on issues that are hard to find